build.js

```diff
@@ -116,6 +118,7 @@ Metalsmith(__dirname)
     writemetadata({
       collections: {
         events: { output: { path: '.events-published.json' }, ignorekeys: ['contents', 'next', 'previous'] },
+        pastevents: { output: { path: '.past-events-published.json' }, ignorekeys: ['contents', 'next', 'previous'] },
```

Comment:
This line won't be necessary, as this metadata files are only used to signal the API which are the newly published jobs and events so it can tweet about them and there's no need to do that with past events.

Reply:
Eliminado